### PR TITLE
DISTMYSQL-272 Release notes for PDPXC 8.0.31

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
new file:   docs/release-notes-pxc-v8.0.31.md
modified:   docs/release-notes.md
modified:   mkdocs-base.yml

### CODE REVIEW

**Review by patrickbirch — APPROVED** [review]
good

File: docs/release-notes-pxc-v8.0.31.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+| :---------------------- | :--------------- |
+| Install instructions    | [Installing Percona Distribution for MySQL](installing.md)|
+
+Percona Distribution for MySQL is the most stable, scalable and secure open-source MySQL distribution, with two download options: one based on Percona Server for MySQL and one based on Percona XtraDB Cluster.
+
+This release is focused on the Percona XtraDB Cluster-based deployment variation. It is based on [Percona XtraDB Cluster 8.0.31-23](https://www.percona.com/doc/percona-xtradb-cluster/8.0/release-notes/8.0.31-23.html)
+
+## Release highlights
+
+This release adds the [GCache encryption and Write-Set cache encryption](https://docs.percona.com/percona-xtradb-cluster/8.0/management/gcache-write-set-cache-encryption.html) feature in [tech preview](./glossary.md#tech-preview).
```
Comment (patrickbirch) [inline docs/release-notes-pxc-v8.0.31.md:13]: Isn't this in 8.0.32?

File: docs/release-notes-pxc-v8.0.31.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+## Supplied components
+
+Review each component’s release notes for What’s new, improvements, or bug fixes. The following is a list of the components supplied with the Percona Server for MySQL-based variation of the Percona Distribution for MySQL:
+
+| Component               | Version   | Description                                |
+| ----------------------- | --------- | -------------------------------------------|
+| Percona XtraBackup      | [8.0.31-24](https://docs.percona.com/percona-xtrabackup/8.0/release-notes/8.0/8.0.31-24.0.html)| An open-source hot backup utility for MySQL-based servers that doesn’t lock your database during the backup.|
+| HAProxy                 | [2.5.12](https://git.haproxy.org/?p=haproxy-2.5.git;a=commit;h=7367a7ceb9494a4c463120bb065d619c8663b00a) | A high-availability and load-balancing solution for Percona XtraDB Cluster. This is a default proxy.|
+| ProxySQL                | [2.4.7](https://docs.percona.com/proxysql/2.4.7.html)| A high performance, high-availability, protocol-aware proxy for MySQL.          |
```
Comment (eleo007) [inline docs/release-notes-pxc-v8.0.31.md]: Can change if we release  ProxySQL 2.4.8 before release date.

**Review by eleo007 — APPROVED** [review]

File: docs/release-notes-pxc-v8.0.31.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/release-notes-pxc-v8.0.31.md:13]: We have it for 8.0.31

File: docs/release-notes-pxc-v8.0.31.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/release-notes-pxc-v8.0.31.md]: Updated

**Pull request merged**